### DISCUSSION TITLE
Hello and thanks

### DISCUSSION THREAD

**Discussion by xpseudonym** [juha-h/baresip-studio#485] [General]
Is this the main discussion forum for the android baresip app?  I'd just like to say - for starters - what a breath of fresh air it is to find a sip client after a such long walk in the desert since csipsimple - thank you

**Comment by juha-h** [comment]
@xpseudonym There  has not been any baresip discussions yet, but thanks for the start and your encouragement.  I too used to be a csipsimple user and when it was not supported anymore, decided to roll my own.
